docs/_releases/1.0.0.md

```diff
@@ -1,5 +1,5 @@
 ---
-version: '1.0'
+version: '1.0.0'
 order: 100
 layout: page
 menu_title: '1.0'
```

Comment:
does this require a change to the menu_title as well?

Reply:
I think the `menu_title` should be ok. It has the correct version format.